pom.xml

```diff
@@ -286,6 +286,9 @@
   <profiles>
     <profile>
       <id>tests</id>
+      <activation>
```

Comment:
I was thinking about that too, but having it in a profile also allows you to quickly skip it with `-P!tests`.